code/src/index.css

```diff
@@ -11,3 +11,148 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.outer-container {
+  width: 100%;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.inner-container {
+  width: 50%;
+  margin: 0 auto;
+  grid-gap: 1rem;
+  gap: 1rem;
+  display: flex;
+  flex-direction: column;
+  padding-top: 1rem;
+  align-items: center;
+  justify-content: center;
+}
+
+.form-parent {
+  width: 50%;
+  background-color: rgb(230, 226, 226);
+}
+
+.form-container {
+  padding: 1rem;
+  border: 1px solid #000;
+  box-shadow: 6px 6px #111;
+  display: flex;
+  flex-direction: column;
+  grid-gap: 1rem;
+}
+
+.input-text {
+  height: 4vw;
+  width: 100%;
+  font-size: 20px;
+}
+
+.submit-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.submit-btn {
+  width: fit-content;
+  padding: 0.5rem;
+  border-radius: 40px;
+  font-family: "Roboto Mono",monospace;
+  border: none;
+  background-color: #fabee3;
+  font-weight: 900;
+  font-size: large;
+}
+
+.header {
+  font-family: "Roboto Mono",monospace;
+}
+
+.sub-header {
+  font-family: "Roboto Mono",monospace;
+}
+
+/* animation to make "happy" text in subheader, change color depending on the 6 second duration */
+
+.h2-animation {
+  font-family: "Roboto Mono",monospace;
+  animation: h2-animation ease;
+  animation-iteration-count: infinite;
+  animation-duration: 6s;
+  animation-fill-mode: both;
+}
+
+@keyframes h2-animation {
+  0% {
+    color: rgb(250, 70, 70);
+  }
+  25% {
+    color: rgb(243, 104, 104);
+  }
+  
+  50% {
+    color: #fd5ec0;
+  }
+  
+  75% {
+    color: #ffade0;
+  }
+
+  100% {
+    color: rgb(243, 104, 104);
+  }
+
+}
+
+.error-msg {
+  color: red;
+  padding-left: 10%;
+}
+
+.thoughts-boxes {
+  padding: 1rem;
+  border: 1px solid #000;
+  box-shadow: 6px 6px #111;
+  display: flex;
+  flex-direction: column;
+}
+
+
+
+.thoughts-container {
+  grid-gap: 1rem;
+  gap: 2rem;
+  display: flex;
+  flex-direction: column;
+  padding-top: 1rem;
+  width: 50%;
+}
+/*word-wrap breaks the line inside box instead of printing it in one line, that continues outside of box.... */
+.thoughts-message {
+  word-wrap: break-word;
+  font-family: "Roboto Mono",monospace;
```

Comment:
Super, these explanations, makes it so much simpler to understand and find  special things and solutions in the code